packages/material-ui-styled-engine-sc/README.md

```diff
@@ -0,0 +1,15 @@
+# @material-ui/styled-engine-sc
+
+This package is a simple wrapper around the `styled-components` package. It is created to be used as alias for the `@material-ui/styled-engine` package, for the developers who would like to use `styled-components` as a styled engine instead of `@emotion/styled`.
+
+## Installation
+
+The installation of the dependency in your package is slightly different from the usual.
+You need to alias the default `emotion` implementation to the `styled-components` one.
+It works with npm and yarn.
```

Comment:
Does this work for npm 6.x? I always thought npm doesn't support aliases.

Reply:
https://docs.npmjs.com/cli-commands/install.html it is documented here, you can search for alias. @oliviertassinari extended this documentation, I thought as well it didn't work with npm, but seems like it is supported at this moment